bakery/src/cli/execute.js

```diff
@@ -388,6 +388,11 @@ const tasks = {
         yargs.positional('slug', {
           describe: 'slug of collection to work on',
           type: 'string'
+        }).option('single', {
```

Comment:
This is super small, but I'll ask anyways 🙂  - I think all of our other options are single character with a one word alias (e.g. `-p` or `--persist`). Is there a reason we're doing `--single` / `--single-book` versus, say, `-s` / `--single` for consistency?

Reply:
Can do